internal/dbusservice/dbusservice.go

```diff
@@ -128,9 +129,18 @@ func parseConfig(cfgPath string) (map[string]map[string]string, error) {
 	for _, section := range iniCfg.Sections() {
 		cfg[section.Name()] = make(map[string]string)
 		for _, key := range section.Keys() {
+			if strings.Contains(key.String(), "<") || strings.Contains(key.String(), ">") {
```

Comment:
I went with the `||` because, as it is a template, it could happen that the user ended up not replacing all the `<>` and the configuration ended up with something like: `actual_issuer.com>` or `<actual_issuer.com`. Does that make sense?

Reply:
Yeah, but it means they edited it, and we are just trying to ensure they at least edited it. If the value is then invalid, so be it and another error will trigger down the line, but they will know where to fix it as they touched it once at least :)